tests/mocha/index.html

```diff
@@ -51,6 +51,7 @@
 	  // Test modules.
 	  'Blockly.test.astNode',
 	  'Blockly.test.blockChangeEvent',
+    'Blockly.test.blockDeleteEvent',
```

Comment:
There appears to be a whitespace error here.  Are there some lurking `\t`s?

Reply:
Lots!